cli/src/cmd_instance.rs

```diff
@@ -268,7 +268,7 @@ impl RunnableCmd for CmdInstanceFromImage {
                             .expect("valid disk name"),
                         size: self.size.clone(),
                     }])
-                    .external_ips(vec![ExternalIpCreate::Ephemeral { pool_name: None }])
+                    .external_ips(vec![ExternalIpCreate::Ephemeral { pool: None }])
```

Comment:
is this now a required parameter?

Reply:
It's not a required parameter, this fixes a breaking field name change as we now take an `Option<NameOrId>`.

Reply:
That makes so much more sense. Sorry for the dumb question. Going too fast!